witnet.toml

```
@@ -68,11 +68,11 @@ enabled = true
 data_request_max_retrievals_per_epoch = 30
 data_request_timeout_milliseconds = 2000
 genesis_path = "genesis_block.json"
-# Enables or disables automatically splitting block mining rewards.
-# If set to `true`, the rewards will be split into multiple outputs of the same value as `collateral_minimum`
-# until the node's set of outputs available for collateralization outnumbers `data_request_max_retrievals_per_epoch`
-# multiplied by `collateral_coinage`, i.e. makes sure it always has spare outputs to collateralize.
-split_mint = true
+# 'mint_pkh' and 'mint_external_percentage' fields allows to the node operator splitting the mint reward
+# between the node address and another address, 'mint_external_percentage' indicates how much percentage
+# is provided to the 'mint_pkh'
+#mint_pkh = "twit1jqgf4rxjrgas3kdhj3t4cr3mg3n33m8zw0aulr"
```

- We can name this as external as well so that both settings are uniform and the user understand that they belong to the same feature.
- We should avoid using the "PKH" word in anything facing to the user and instead call them "addresses".
```suggestion
#mint_external_address = "twit1jqgf4rxjrgas3kdhj3t4cr3mg3n33m8zw0aulr"
```